keyboards/eason/greatsword80/greatsword80.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2023 Eason <xiaoxuxk@yeah.net>
```

Comment:
This file should be removed, assuming the `matrix` data in `info.json` file is correct.